### PR TITLE
HTTPS for asb route

### PR DESCRIPTION
**Pull request by shawn-hurley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ configurations.
 * `make vendor`: Installs or updates the dependencies
 * `make build`: Builds the binary from source
 * `make install`: Installs the built binary.
-* `make run`: Runs the broker with the default profile, configured via `/etc/dev.config.yaml`
+* `make prepare-local-env`: will set up the local environemt to test and run the broker against a local deployment of [catasb](https://github.com/fusor/catasb)
+* `make run`: Runs the broker with the default profile, configured via `/etc/generated_local_development.yaml`
```
Comment (jmrodri) [inline README.md]: it's not `/etc/` it's just the `etc` in the project repo.

**Comment by shawn-hurley** [comment]
Need to update catasb and merge requests together.

**Comment by dymurray** [comment]
[(8) Create an Adapter Registry in Broker to Communicate with AtomicRegistry portion of RHCC (ISV Images)](https://trello.com/c/TuUsP9bL/303-8-create-an-adapter-registry-in-broker-to-communicate-with-atomicregistry-portion-of-rhcc-isv-images)

**Comment by johnkim76** [comment]
[(5) Configure https with Broker's route.](https://trello.com/c/DngTXbRR/324-5-configure-https-with-brokers-route)

File: etc/example-config.yaml [hunk 2]
```diff
@@ -35,3 +35,5 @@ broker:
   launch_apb_on_bind: false
   recovery: true
   output_request: true
+  ssl_cert_key: /path/to/key
```
Comment (rthallisey) [inline etc/example-config.yaml]: Should we default this to ```""```? dev_broker is being set to true, which means this should get defaulted to an empty string.

File: scripts/run_local.sh [hunk 3]
```diff
@@ -38,5 +38,14 @@ if [ -z "${BROKER_CONFIG}" ]; then
   exit 1
 fi
 
-echo "Running ${BROKER_CMD} --config ${BROKER_CONFIG}"
-${BROKER_CMD} --config ${BROKER_CONFIG}
+if [ "${BROKER_INSECURE}" == "true" ]; then
+  echo "${BROKER_INSECURE}"
```
Comment (rthallisey) [inline scripts/run_local.sh]: What's the point of this? Why not do the check ```"${BROKER_INSECURE}" == "true"``` below?

File: etc/example-config.yaml (hunk 2 above)
Comment (shawn-hurley) [inline etc/example-config.yaml]: what does this have to do with the dev_broker? I would rather never show this as being able to accept an empty string as that could be confusing.

Comment (rthallisey) [inline etc/example-config.yaml]: IIRC when we're using the broker in dev mode we should skip ssl and run insecure.  But it looks like you covered that with the insecure flag so really what goes here won't matter if we use that flag.
